docs/monthly-meeting/2022-06.md

```diff
@@ -0,0 +1,175 @@
+
+- [ ] Creating a 'docs-approved' (wording to be agreed) label for CPython.
+
+Needs discussion, please open issue on core-workflow repo.
+(There's a template for label changes: https://github.com/python/core-workflow/issues/new/choose)
+
+
+### 'Internal' items
+
+*For and about the Community or Working Group*
+
+#### Docs WG renaming to Editorial Board
+
+> If we rename the WG to an Editorial Board, we should clarify the interpretation of the name.
+>
+> A quarterly meeting for the WG seems like the right cadence.
+> 
+> Editorial Board is the agreed name. Wikipedia has a good definition which aligns with our meaning.
+
+Status: There is currently no ceremony/meetings for the work group/editorial board, to meet and make progress on this.
+
+- [ ] Petr: Do the rename in all the docs
+
+##### Docs governance
+
+Guido: There may be a need for a bit more leadership within this group/"Community". Having an organiser for the meetings.
+
+Petr: What does this mean in practice?
+
+Guido: Being in charge of the agenda, how long things are discussed.
+
+Petr: Fine with that, that's how things work today. Does this need to be formalised somehow?
+
+Guido: Someone should have 'Convening' power -- someone to take ownership of filling the agenda, managing the admin, etc.
+
+Adam: Typing has more active solicitation of agenda items. That might be a concrete improvement/action item out of this.
+
+Petr: *in agreement*
+
+CAM: The Documentation category on Discourse serves this role, and we have posts on there for some of these meetings.
+
+Ned: Where are decisions made? Typing-sig makes decisions affecting them, as it is a forum of typing-checker implementers and users. Can Discourse be a place decisions are made?
+
+Ned: There's agreement about *that* we should use Diataxis, but not about *how* we should implement it.
+
+- [ ] Petr: Solicit agenda items on Discourse, for the next/future meetings
+
+### Python Project Documentation
+
+*Relating to `docs.python.org`, `peps.python.org`, `devguide.python.org`, &c.*
+
+#### Proofreaders GitHub team
+
+It would be useful to have a GitHub team that can be `@mentioned` in docs PRs to request reviews of the English grammar.
+
+- [ ] Petr: Create a `proofreaders` team (Initially: CAM, Adam, Pradyun, Hugo), post about this team on Discourse (or similar) for a wider audience
+
+#### Devguide improvement plans
+
+Ezio made some comments in [a thread](https://discord.com/channels/935215565872693329/980818577559007242):
+
+> The first plan is to overhaul a bit the GitHub docs, and is also related to other changes about labels.
+The second plan is still somewhat half-baked, but I wanted to
+> 1. add some instructions about following the development of CPython
+
+Ezio: Add a changelog page to the devguide for an overview of workflow changes
+
+- [ ] Ezio: Add a “what's new” section to the devguide (key/important changes)
```

Comment:
I created https://github.com/python/devguide/issues/885

Reply:
I've checked it in Past Action Items in the new agenda.